### PR TITLE
Updated test data paths for busco/busco

### PR DESCRIPTION
**Pull request by GallVp**
## PR checklist

Closes #XXX 

- [x] Updated test data paths for busco/busco
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/busco/busco/tests/main.nf.test [hunk 1]
```diff
@@ -250,8 +248,6 @@ nextflow_process {
 
             with(path("${process.out.busco_dir[0][1]}/logs/busco.log").text) {
                 assert contains('DEBUG:busco.run_BUSCO')
-                assert contains("'use_augustus', 'True'")
-                assert contains("'use_metaeuk', 'False'") // AUGUSTUS
```
Comment (fellen31) [inline modules/nf-core/busco/busco/tests/main.nf.test:254]: And this?

File: modules/nf-core/busco/busco/tests/main.nf.test [hunk 2]
```diff
@@ -208,8 +208,6 @@ nextflow_process {
 
             with(path("${process.out.busco_dir[0][1]}/logs/busco.log").text) {
                 assert contains('DEBUG:busco.run_BUSCO')
-                assert contains("'use_augustus', 'False'")
-                assert contains("'use_metaeuk', 'True'") // METAEUK
```
[the remark this answers is not in the document]
Comment (GallVp) [reply modules/nf-core/busco/busco/tests/main.nf.test:212]: This is not stable under the conda profile. I have been testing it elsewhere. Please see: https://github.com/GallVp/nxf-components/actions/runs/10102555955/job/27938374438

File: modules/nf-core/busco/busco/tests/main.nf.test (hunk 1 above)
Comment (GallVp) [reply modules/nf-core/busco/busco/tests/main.nf.test:254]: Same as above.

File: modules/nf-core/busco/busco/tests/main.nf.test (hunk 2 above)
[the remark this answers is not in the document]
Comment (fellen31) [reply modules/nf-core/busco/busco/tests/main.nf.test:212]: Alright!

**Review by fellen31 — APPROVED** [review]

**Pull request merged**